### ISSUE TITLE
Multiple users can create account with the same email

### ISSUE THREAD

**Issue by prondubuisi** [toriboi/LR#3]
I would like to work on this issue if confirmed

**Comment by toritsejuFO** [comment]
Yes, noticed, this wouldn't be possible though if the unique attribute is set on the database level. Although, that would require more error handling. Nice observation thanks. Fix away :grin:
